Sources/Plugins/AttachmentManager/AttachmentManager.swift

```diff
@@ -121,6 +121,12 @@ open class AttachmentManager: NSObject, InputPlugin {
         })
     }
     
+    /// Performs an animated removal of all attachments
+    open func removeAllAttachments() {
+        self.attachments.removeAll()
```

Comment:
I think this should also call delegate methods the same as it does for removing one attachment?